test/integration/fixture_image_simple_test.go

```diff
@@ -263,18 +266,20 @@ func assertImageSimpleSquashedTrees(t *testing.T, i *image.Image) {
 }
 
 func assertImageSimpleTrees(t *testing.T, i *image.Image) {
-	t.Log("Asserting trees...")
-	one := filetree.NewFileTree()
+	t.Helper()
+	//t.Log("Asserting trees...")
```

Comment:
nit: t.Log 

Reading more of these - are we keeping these for the developer's use case to uncomment debugging test

Reply:
I think I'll keep it for now